conf/client.conf

```diff
@@ -67,3 +67,8 @@ tlsTrustStorePath=
 
 # TLS TrustStore password
 tlsTrustStorePassword=
+
+# Set up TLS provider for web service
+# When TLS authentication with CACert is used, the valid value is either OPENSSL or JDK.
+# When TLS authentication with KeyStore is used, available options can be SunJSSE, Conscrypt and so on.
+webserviceTlsProvider=
```

Comment:
do we really need this config? seems it's just the same as `--tls-provider`

Reply:
You are right, the `webserviceTlsProvider` equals with `--tls-provider` flag. I suggest we should add this config.